samples/getting-started/simulation/README.md

```diff
@@ -0,0 +1,34 @@
+---
+page_type: sample
+languages:
+- qsharp
+products:
+- qdk
+- azure-quantum
+description: "These samples demonstrate how to use the different types of quantum simulation"
+---
+
+# Running the Simulation
+
+## Prerequisites
+
+- The Microsoft [Quantum Development Kit](https://docs.microsoft.com/azure/quantum/install-overview-qdk/).
+- Jupyter Notebook
+
+## LargeSimulation.ipynb
```

Comment:
It may be best to follow the format for other sample README.md files and have this under a `## Manifest` section?